Code/Framework/AzCore/AzCore/RTTI/TypeInfo.h

```diff
@@ -167,40 +174,46 @@ namespace AZ
             static const AZ::TypeId s_uuid = AZ::TypeId::CreateNull();
             return s_uuid;
         }
-
+#if defined(AZ_COMPILER_MSVC)
+        // There is a bug with the MSVC compiler when using the 'auto' keyword here. It appears that MSVC is unable to distinguish between a template
+        // template argument with a type variadic pack vs a template plate argument with a non-type auto variadic pack. 
```

Comment:
repeat: typo with "template plate"

```suggestion
        // template argument with a type variadic pack vs a template template argument with a non-type auto variadic pack. 
```